### PR TITLE
VFX Graph sample github button in package manager

### PR DESCRIPTION
**Pull request by MarieGuff**
### Samples github link button
---
### Purpose of this PR
Better access for the user to the VFX Graph samples project

Added a link to the github (public)repo of Visual Effect Graph Samples in the package manager. This link forward to the man page of the git repo, because the release page doesn't benefit from the visuals of the samples and the description of the project. 

image

Added file: com.unity.visualeffectgraph\Editor\SamplesLinkPackageManagerExtension

Copied and modified from: com.unity.terrain-tools\Editor\SamplesLinkPackageManagerExtension

---
### Testing status
Tested if button worked
Tested button being hidden in other packages
Tested button still available on Terrain Tool 
Tested button disabled on package deinstallation (tested using shadergraph test project)

### CODE REVIEW

File: com.unity.visualeffectgraph/Editor/SamplesLinkPackageManagerExtension.cs.meta [hunk 1]
```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: 4ae080a8a6a24a9418d6cfc4f4ad403f
```
Comment (SeanPuller) [inline com.unity.visualeffectgraph/Editor/SamplesLinkPackageManagerExtension.cs.meta:2]: this guid is the same as the guid for the meta file in the terrain tools package. it needs to be updated and pushed, one should be generated by default

**Review by SeanPuller — CHANGES_REQUESTED** [review]

**Comment by VladNeykov** [comment]
Hi, since Vita is out just dropping a few UX-y comments:
- The samples are mainly HDRP, it might be a bit misleading for URP users to have that button without more context.
- **Download** implies that the button will download the samples, but it's just an external link. 
- One button for one link might not scale well. Wonder if a resources doc page on our manual where we can add various link (samples, spaceship demo, learning materials, that Scenario Week vfx samples project, etc.) might not be a better place? Maybe somewhere here on top:
image
- We already have a link to the samples from within the VFX Graph. No objections for having it in more places, just FYI:
Unity_jIKWuOekDV

**Comment by MarieGuff** [comment]
> Hi, since Vita is out just dropping a few UX-y comments:
> 
> * The samples are mainly HDRP, it might be a bit misleading for URP users to have that button without more context.
> * **Download** implies that the button will download the samples, but it's just an external link.
> * One button for one link might not scale well. Wonder if a resources doc page on our manual where we can add various link (samples, spaceship demo, learning materials, that Scenario Week vfx samples project, etc.) might not be a better place? Maybe somewhere here on top:
>   image
> * We already have a link to the samples from within the VFX Graph. No objections for having it in more places, just FYI:
>   Unity_jIKWuOekDV

Thanks I will modify the button name
Unfortunately I haven't found any script/documentation/whatever to add a link somewhere else but i'll see if I find something (maybe doc team can help me with that?)
I will also try to find how to add a tooltip for more context, but otherwise i'll use the "HDRP Samples github page" name for the button 
I will start gathering ressources links and see with mathieu how to present it to make a global ressource page in a next update

**Review by SeanPuller — APPROVED** [review]

**Comment by MarieGuff** [comment]
Future improvements:

- Combined link for several projects
- Button UI revision

**Pull request merged**